### ISSUE TITLE
CreateElement("script") not recognized?

### ISSUE THREAD

**Issue by LeMoussel** [AngleSharp/AngleSharp#208]
Inserting a script element, seems not to be recognized.
With this code `jsGetTitle1` is OK but `jsGetTitle2` is undefined. However, insertion of the script element has been successfully completed.

``` C#
            string sourceHTML = @"<!doctype html>
                <html>
                <head><title>AngleSharp ScriptingService Sample</title></head>
                <body>
                    <div id='full-name' style='display:inline-block;font-size:40px;line-height:40px;margin:10px auto 0 auto;word-wrap:none;white-space:nowrap;'>some stuff...</div>
                    <script>    
                        function getT1() { return document.title; }      

                        var node = document.getElementById('full-name');
                        document.writeln(node.id + ' ' + node.style.fontSize);
                    </script>
                </body>
                </html>";

            ScriptingService javascriptService = new ScriptingService();
            IConfiguration config = Configuration.Default.WithDefaultLoader().With(javascriptService).WithCss();

            IDocument ashDocument = await BrowsingContext.New(config).OpenAsync(m => m.Content(sourceHTML));

            IElement elementJSScript = ashDocument.CreateElement("script");
            elementJSScript.TextContent = "function getT2() { return document.title; }";
            ashDocument.Body.AppendChild(elementJSScript);

            Console.WriteLine(ashDocument.DocumentElement.OuterHtml);

            JsValue jsGetTitle1 = javascriptService.Engine.GetJint(ashDocument).GetValue("getT1");
            string documentTitle1 = jsGetTitle1.Invoke().AsString();
            Console.WriteLine("T1 Document Title: {0}", documentTitle1);

            JsValue jsGetTitle2 = javascriptService.Engine.GetJint(ashDocument).GetValue("getT2");
            string documentTitle2 = jsGetTitle2.Invoke().AsString();
            Console.WriteLine("T2 Document Title: {0}", documentTitle2);


```

**Comment by FlorianRappl** [comment]
This is known behavior that will be fixed in one of the next version (I hope with v0.9.5, but for sure with v0.9.6). Scripts don't start right away, but need to be started. In a scenario involving the parser it is done automatically, but if inserted manually the required checks and regulated logic is missing.

I'll mark it as bug (since it is unexpected behavior atm) and enhancement (as it would be desired). I'll also mark it for v0.9.6, even though I'll try to make it available for v0.9.5.

**Comment by LeMoussel** [comment]
An idea when will be available v0.9.5?

**Comment by FlorianRappl** [comment]
Hopefully, when now everything works out according to my schedule ~6th of January. Don't know exactly though (it depends on how many other things with higher priority occur in the next weeks).
